### ISSUE TITLE
use LV2 Atom instead of LV2 Event

### ISSUE THREAD

**Issue by LadyNamedLaura** [jeremysalwen/TalentedHack#7]
LV2 Event has been deprecated for a while and it seems software is dropping support for it (I can't get this plugin to work under ardour because it can't figure out the midi ports)

```
LV2 Category: 'Pitch Shifter Plugin'
Port 0 ('MIDI Out') has no known data type
Port 1 ('MIDI In') has no known data type
```

**Comment by jeremysalwen** [comment]
I would be happy to accept a patch porting to LV2 Atom.

**Comment by AS383** [comment]
Ardour 6.9.0 has the same issue.

**Comment by michal-cab** [comment]
ardour 6.9 is not blacklisting plugin now but when trying to load it it ends with seg.fault :(.
unfortunately i am not able to dig deeper info with gdb/console output ... i am @ubuntustudio 20.04

**Comment by jeremysalwen** [comment]
Ah, turns out the issue was a bug I had introduced which crashed if the midi_out port was not connected.  Should be fixed now in v1.92.  I tested it in Ardour6.5.0.
